cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -347,6 +347,25 @@ describe(["@tier1"], "Source Analysis", () => {
         application.verifyEffort(this.analysisData["tackle-testapp-public-4-targets"]["effort"]);
     });
 
+    // Automates customer bug MTA-2973
+    it(["@tier0"], "Source analysis on tackle app public with custom rule", function () {
```

Comment:
Is this meant to be a tier0 test ? 
We already have a tier0 test for analysis  in source_analysis_without_creds.test.ts .

Reply:
Hi @nachandr, Yes, this test case is intended to be tier0. I noticed that there are tests tagged as tier1 in source_analysis_without_creds.test.ts. I might be missing something—could you please point me in the right TC?

Reply:
This is a closed-loop ticket and intended to be marked as tier0 as mentioned in the Automation ticket attached above.